<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>62</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>0</y>
         <width>81</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Serial Port:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="ports">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>4</y>
         <width>82</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>168</x>
         <y>0</y>
         <width>69</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Baud Rate:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="baudrate">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>4</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>328</x>
         <y>1</y>
         <width>61</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Data Bits:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="data_bits">
       <property name="geometry">
        <rect>
         <x>392</x>
         <y>4</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>4</y>
         <width>45</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Parit√§t:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="connect">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>32</y>
         <width>73</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QPushButton" name="disconnect">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>84</x>
         <y>32</y>
         <width>73</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
      <widget class="QComboBox" name="parity">
       <property name="geometry">
        <rect>
         <x>488</x>
         <y>4</y>
         <width>69</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>564</x>
         <y>4</y>
         <width>57</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop Bits:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="stop_bits">
       <property name="geometry">
        <rect>
         <x>624</x>
         <y>4</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>672</x>
         <y>4</y>
         <width>97</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Fluss Controlle:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="flow_control">
       <property name="geometry">
        <rect>
         <x>768</x>
         <y>4</y>
         <width>89</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="led_count">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>35</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>LED Count:</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="serial_reciver_out">
        <property name="font">
         <font>
          <family>Noto Mono</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTableWidget" name="led_table">
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="TestButton">
        <property name="text">
         <string>Test Button</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="all_leds_off">
        <property name="text">
         <string>All LED's off</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
